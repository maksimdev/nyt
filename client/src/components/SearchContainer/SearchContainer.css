.resultWrapper {
  margin-top: 20px;
  padding: 5%;
  width: 70%;
  visibility: hidden;
}

.visible {
  visibility: visible;
}

.notFound {
    color: purple;
}