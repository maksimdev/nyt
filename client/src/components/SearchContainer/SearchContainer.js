import React from 'react';
import SearchPanel from './SearchPanel'
import './SearchContainer.css';

export function SearchContainer() {
    return (
      <SearchPanel />
    );
};