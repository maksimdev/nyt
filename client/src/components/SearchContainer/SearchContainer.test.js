import React from 'react';
import * as Enzyme from 'enzyme';
import SearchContainer from './SearchContainer';
import Adapter from 'enzyme-adapter-react-16';
import enzymeSerializer from 'enzyme-to-json/serializer';

Enzyme.configure({ adapter: new Adapter() })

expect.addSnapshotSerializer(enzymeSerializer);
describe('SearchContainer', () => {
    it('should to match snapshot', () => {
        const SearchContainerComponent = Enzyme.shallow(<SearchContainer />);
        expect(SearchContainerComponent).toMatchSnapshot();
    });
});