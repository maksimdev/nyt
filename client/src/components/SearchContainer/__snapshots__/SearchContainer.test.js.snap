// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchContainer should to match snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
